app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(list_of_player: list[Player]) -> int:
+    return sum(rate.get_rating() for rate in list_of_player)
```

Comment:
why do u call variable rate? here will be an instance of Player class inside this variable